packages/samples/testserver/body-boolean/body-boolean.cadl

```diff
@@ -22,7 +22,9 @@ namespace BoolsTrue {
   @doc("Set Boolean value true")
   @operationId("bool_putTrue")
   @put
-  op setTrue(@body value: true): OkResponse<{}> | Error;
+  op setTrue(@body value: true): {
```

Comment:
Why use this instead of just `Response<200>` ?

Reply:
Good question. I tried `Response<200>` and the result was that the response description was "Http response with status code" which seemed not great. But I admit that this is just a hack and we should figure out how to get the right description coming out in a more natural way.  I'd like to defer that -- and some other design changes I think we'll want to a later PR if that's okay.